blog/content/post/2019-04-18-USB-sniffing-with-OpenVizsla.md

```diff
@@ -0,0 +1,195 @@
+
+These are frames from connected USB stick, that does absolutely nothing.
+The last square bracket represents number of bytes send, after that you can
+read a packet identifier. `IN` mean a `TOKEN` `IN` signal.
+`ACK` is a `handshake` signal, which you can observe in examples above.
+
+Now, lets connect to OpenVisla some real thing. Our choice was a stick with
+live system, which should try to be recognized by the PC.
+
+After a second we've had:
+
+```
+[        ]   1.836409 d=  0.000000 [135.1 + 39.783] [ 16] DATA0: 55 53 42 53 60 00 00 00 00 00 00 00 00 f1 b0
+[        ]   1.836634 d=  0.000000 [135.3 + 14.400] [ 34] DATA0: 55 53 42 43 61 00 00 00 00 00 00 00 00 00 06 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 aa 90
+[        ]   1.836790 d=  0.000000 [135.4 + 45.050] [ 16] DATA1: 55 53 42 53 61 00 00 00 00 00 00 00 00 fc 20
+[        ]   1.836985 d=  0.000000 [135.5 +114.817] [ 34] DATA1: 55 53 42 43 62 00 00 00 00 10 00 00 80 00 0a 28 00 00 00 00 00 00 00 08 00 00 00 00 00 00 00 49 3f
+[        ]   1.837871 d=  0.000000 [136.5 +  1.033] [515] DATA0: eb 3c 00 00 00 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 00 00 00 00 12 00 02 00 00 00 00 00 00 00 00 00 00 16 1f 66 6a 00 51 50 06 53 31 c0 88 f0 50 6a 10 89 e5 e8 c0 00 8d 66 10 cb fc 31 c9 8e c1 8e d9 8e d1 bc 00 7c 89 e6 bf 00 07 fe c5 f3 a5 be ee 7d 80 fa 80 72 2c b6 01 e8 60 00 b9 01 00 be be 8d b6 01 80 7c 04 a5 75 07 e3 19 f6 04 80 75 14 83 c6 10 fe c6 80 fe 05 72 e9 49 e3 e1 be a2 7d eb 4b 31 d2 89 16 00 09 b6 10 e8 2e 00 bb 00 90 8b 77 0a 01 de bf 00 c0 b9 00 ae 29 f1 f3 a4 fa 49 74 14 e4 64 a8 02 75 f7 b0 d1 e6 64 e4 64 a8 02 75 fa b0 df e6 60 fb e9 50 13 bb 00 8c 8b 44 08 8b 4c 0a 0e e8 5a ff 73 2a be 9d 7d e8 1c 00 be a7 7d e8 16 00 30 e4 cd 16 c7 06 72 04 34 12 ea f0 ff 00 f0 bb 07 00 b4 0e cd 10 ac 84 c0 75 f4 b4 01 f9 c3 2e f6 06 b0 08 80 74 22 80 fa 80 72 1d bb aa 55 52 b4 41 cd 13 5a 72 12 81 fb 55 aa 75 0c f6 c1 01 74 07 89 ee b4 42 cd 13 c3 52 b4 08 cd 13 88 f5 5a 72 cb 80 e1 3f 74 c3 fa 66 8b 46 08 52 66 0f b6 d9 66 31 d2 66 f7 f3 88 eb 88 d5 43 30 d2 66 f7 f3 88 d7 5a 66 3d ff 03 00 00 fb 77 9d 86 c4 c0 c8 02 08 e8 40 91 88 fe 28 e0 8a 66 02 38 e0 72 02 b0 01 bf 05 00 c4 5e 04 50 b4 02 cd 13 5b 73 0a 4f 74 1c 30 e4 cd 13 93 eb eb 0f b6 c3 01 46 08 73 03 ff 46 0a d0 e3 00 5e 05 28 46 02 77 88 c3 52 65 61 64 00 42 6f 6f 74 00 20 65 72 72 6f 72 0d 0a 00 80 90 90 90 90 90 90 90 90 90 90 90 90 90 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 80 00 01 00 a5 fe ff ff 00 00 00 00 50 c3 00 00 55 aa b8 c4
+```
+
+Speed of data transfer with all the logged information was such, that log file
+we wanted to be created (plain text) was 200MB in a few seconds.
+
+We had to interrupt it, but all the computer memory was busy making logs.
+Only way to stop it was to disconnect USB device manually. 
+
+## Summary
+
+Summary of the post.
+
+OPTIONAL ending (may be based on post content):
```

Comment:
@lwcislo whole blog post is great reading. It end in little bit unexpected way. I think in summary we should provide some promise of future work an it would be good enough to publish.